spec/models/user_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+
+  describe "callbacks" do
+    it "downcase email before saving" do
+      user = FactoryBot.create(:user)
```

Comment:
Thay vì tạo như này thì anh dùng let! tạo ra 1 object r dùng ở mỗi khối it đc ko em

Reply:
đoạn dòng 6 7 này thì như vậy được ạ. nhưng đoạn từ 16 trở đi thì em dùng let, ví mỗi context của em là một lỗi valid khác nhau, nên em ko để let!